### PR TITLE
Add List.chunkBySize documentation

### PR DESCRIPTION
**Pull request by Banashek**
Preliminary pull-request to make sure I've got the formatting and such correct.

I'll add in the Array and Seq implementations as well if desired.

### CODE REVIEW

**Comment by Banashek** [comment]
As a note. I saw that the contributing.md says to send PRs to live, but the last accepted pr was sent to master. I'm new to PRs in general, and was wondering if it was preferred for me to make a branch for each PR and send the PR to live or stay with master?

**Comment by dend** [comment]
@Banashek typically you want them into `live` because that way your changes will propagate to production right away. That said, I am doing re-syncs constantly, so even if you submit against `master`, I will sync the branches later.

File: docs/conceptual/list.chunkbysize['t]-function-[fsharp].md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+Type: **'T** [list](https://msdn.microsoft.com/library/c627b668-477b-4409-91ed-06d7f1b3e4a7)
+
+
+The input list.
+
+## Exceptions
+
+|Exception|Condition|
+|----|----|
+|[ArgumentException](https://msdn.microsoft.com/library/system.argumentexception.aspx)|Thrown when chunkSize is not positive.|
```
Comment (dend) [inline docs/conceptual/list.chunkbysize['t]-function-[fsharp].md]: Wrap `chunkSize` in backticks.

**Comment by dend** [comment]
Overall the PR looks good, just needs a minor tweak. LGTM 🔥 

Once someone from the PG validates the content, this will auto-merge.

File: docs/conceptual/list.chunkbysize['t]-function-[fsharp].md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+**Namespace/Module Path:** Microsoft.FSharp.Collections.List
+
+**Assembly:** FSharp.Core (in FSharp.Core.dll)
+
+
+## Syntax
+
+```fsharp
+// Signature:
+List.chunkBySize : int -> 'T list -> 'T list list
```
Comment (cartermp) [inline docs/conceptual/list.chunkbysize['t]-function-[fsharp].md]: Let's match the exact signature in [`list.fsi`](https://github.com/Microsoft/visualfsharp/blob/master/src/fsharp/FSharp.Core/list.fsi#L70):

```fs
List.chunkBySize: chunkSize:int -> list: 'T list -> 'T list list
```

File: docs/conceptual/list.chunkbysize['t]-function-[fsharp].md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+// Usage:
+List.chunkBySize chunkSize list
+```
+
+#### Parameters
+*chunkSize*
+Type: [int](https://msdn.microsoft.com/library/025d5455-3622-4ea5-9573-3ecbd4ee1375)
+
```
Comment (cartermp) [inline docs/conceptual/list.chunkbysize['t]-function-[fsharp].md]: Nitpick: extraneous space here; let's just have a single line break when separating things.

**Comment by cartermp** [comment]
Thanks, @Banashek!  Aside from my nitpick feedback, this also looks good.

File: docs/conceptual/list.chunkbysize['t]-function-[fsharp].md (hunk 3 above)
Comment (Banashek) [inline docs/conceptual/list.chunkbysize['t]-function-[fsharp].md]: Just to check: single line breaks throughout the whole file?

Comment (dend) [inline docs/conceptual/list.chunkbysize['t]-function-[fsharp].md]: Yup

Comment (Banashek) [inline docs/conceptual/list.chunkbysize['t]-function-[fsharp].md]: I added single line breaks before and after each heading, and reduced the double breaks to single breaks.

File: docs/conceptual/list.chunkbysize['t]-function-[fsharp].md (hunk 2 above)
Comment (Banashek) [inline docs/conceptual/list.chunkbysize['t]-function-[fsharp].md]: This makes sense. I was following other documentation, but I agree that signatures should be copy-pasted from the .fsi files.

**Comment by cartermp** [comment]
LGTM 🚢 🇮🇹

**Pull request merged**